main_net_config.conf

```diff
@@ -464,20 +578,30 @@ block = {
   proposalExpireTime = 259200000 // 3 day: 259200000(ms)
 }
 
-# Transaction reference block, default is "head", configure to "solid" can avoid TaPos error
-trx.reference.block = "solid" // head;solid;
+# Transaction reference block, default is "solid", configure to "head" may accur TaPos error
+# trx.reference.block = "solid" // head;solid;
 
 # This property sets the number of milliseconds after the creation of the transaction that is expired, default value is  60000.
 # trx.expiration.timeInMilliseconds = 60000
 
 vm = {
   supportConstant = false
+  maxEnergyLimitForConstant = 100000000
   minTimeRatio = 0.0
   maxTimeRatio = 5.0
   saveInternalTx = false
 
+  # Indicates whether the node stores featured internal transactions, such as freeze, vote and so on
+  # saveFeaturedInternalTx = false
+
   # In rare cases, transactions that will be within the specified maximum execution time (default 10(ms)) are re-executed and packaged
   # longRunningTime = 10
```

Comment:
`longRunningTime` for what ?

Reply:
The purpose of this configuration is to specify the maximum execution time for a contract transaction, as described in .
This configuration is being used temporarily due to a feature adjustment.